### PR TITLE
Create SDK introduction, Closes #205, #233

### LINKED ISSUE #205: Create SDK introduction
- Installation of SDK
- Verify installation by starting a white-label blockchain application

Related issue: LiskHQ/lisk-sdk/issues/3162

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #205, #233

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+| ---------------- | ------- |
+| Node.js          | 10.15.3 |
+| PostgreSQL       | 10+     |
+| Redis (optional) | 5+      |
+
+You can find further details on installing these dependencies in our [pre-installation setup guide](framework/setup.md#pre-install).
+
+### Installation of the Lisk Framework
+
+To start using the Lisk SDK you need to install one NPM package - [lisk-framework](https://www.npmjs.com/package/lisk-framework):
```
Comment (MaciejBaj) [inline introduction.md]: This is outdated installation instructions, also please think of the basic "steps" of running an app using Lisk SDK.

This section should be divided:

using Alpha SDK
- install Alpha SDK
- run your first app 
- the next steps of developing on Alpha SDK

contributing to Alpha SDK 
- compile the following packages - Elements, Framework, SDK 
- Linking the packages for the development ease (please ask @ishantiw for the more details)

**Review by MaciejBaj — CHANGES_REQUESTED** [review]

File: configuration.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+## Start with default values
+
+```js
+const app = new Application(genesisBlockDevnet); // uses predefined values, see full list below
+```
+
+## List of configuration options
+
+Here is a complete list of all available configuration options of the Lisk SDK.
```
Comment (RachBLondon) [inline configuration.md]: Here is a complete list of all *the* available configuration options of the Lisk SDK.

File: configuration.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+The values below are the default values. They will be used automatically when starting the application like described in [start with default values](#start-with-default-values).
```
Comment (RachBLondon) [inline configuration.md]: application *as* described

File: configuration.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+                }
+            ]
+		}
+	}
+}); 
+```
+
+## Constants
+
+Inside of `app.genesisConfig`, specific constants for the blockchain application are set.
```
Comment (RachBLondon) [inline configuration.md]: Inside `app.genesisConfig` specific constants for the blockchain application are set.

File: introduction.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+## Disclaimer - Mostly Harmless
+
+Please read our disclaimer carefully. The current version of the Lisk SDK is the alpha release of the Lisk SDK.
+We have released the Lisk SDK in its current form in order for us to improve the development experience through community feedback and contributions.
+
+We strictly discourage anyone from using the alpha release of the Lisk SDK for any production-based blockchain applications, i.e. a blockchain operating on a real mainnet.
+Over the course of the alpha phase there will be significant changes in the Lisk protocol and implementation, which will eventually bring the accessibility and reliability to a level which is feasible for production-based blockchain applications.
+At this time we only recommend the Lisk SDK for proof-of-concept blockchain applications, i.e. a blockchain operating on a fake testnet.
+
```
Comment (RachBLondon) [inline introduction.md:17]: Should we add a specific message to indicate that the genesis creation and delegate accounts are not secure with this release? @Tschakki @MaciejBaj

File: introduction.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+## What is the Lisk SDK
+
+The Lisk SDK aims to provide an easy and reliable software development kit for building blockchain applications which are compatible with the [Lisk Protocol](https://lisk.io/documentation/lisk-protocol).
+The architecture of the Lisk SDK has been designed so that it can be extended to meet the requirements of a wide variety of blockchain application use-cases.
+The codebase is written entirely in JavaScript, which means for a majority of developers, no significant change of tools or mindset is required to get started.
+The Lisk SDK makes every effort to allow developers to focus simply and purely on writing the code that matters to their own blockchain application, and nothing more.
+
+### Architecture Overview
+
+The Lisk SDK operates on the NodeJS runtime and consists primarily of an application framework (Lisk Framework), a collection of libraries providing blockchain application functionalities (Lisk Elements), and a powerful command-line tool (Lisk Commander) allowing developers to manage a Lisk node instance and interact with a Lisk compatible network.
```
Comment (RachBLondon) [inline introduction.md]: tool (Lisk Commander) *which allows* developers to manage a Lisk node instance and interact with a Lisk compatible network.

File: introduction.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+The diagram below provides a high-level overview of the architecture:
+
+![Diagram](assets/diagram_sdk.png)
+
+## Package Directories
+
+| Directory                                   | Description                                                                                                                                          |
+| ------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Framework](lisk-framework/introduction.md) | The application framework is establishing and maintaining the interactions between the modules of a Lisk blockchain application.                     |
```
Comment (RachBLondon) [inline introduction.md]: The application framework *establishes and maintains* the interactions between the modules of a Lisk blockchain application.

File: introduction.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+| [Elements](lisk-elements/introduction.md)   | A collection of libraries, each of them implementing some form of blockchain application functionality such as cryptography, transactions, p2p, etc. |
+| [Commander](lisk-commander/introduction.md) | A command line tool allowing developers to manage a Lisk node instance and interact with a Lisk compatible network.                                  |
```
Comment (RachBLondon) [inline introduction.md]: A command line tool *which allows* developers to manage a Lisk node instance and interact with a Lisk compatible network.

File: introduction.md [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+To install the `alpha` version of the NPM package [lisk-sdk](https://www.npmjs.com/package/lisk-sdk), run:
+
+```bash
+npm install lisk-sdk@alpha # install lisk-sdk alpha version 
+npm install --save lisk-sdk@alpha # add --save flag to save it to package.json
+```
+
+### Usage
+
+First steps to start developing your blockchain application:
```
Comment (RachBLondon) [inline introduction.md]: *The* first steps to start developing your blockchain application:

File: introduction.md [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+mkdir my-app # create the root folder for your blockchain application
+cd my-app # navigate into the root folder
+npm init # initialize your package.json
+npm install --save lisk-sdk@alpha # install lisk-sdk alpha version and save it to package.json
+touch index.js # create the index file to start the application
+```
+
+Inside of `index.js`, require the `lisk-sdk` package to create and start the application.
+
+Below you see the minimal version of `index.js` that is needed to successfully kick-start the blockchain application:
```
Comment (RachBLondon) [inline introduction.md]: Below *is* the minimal version of `index.js` that is needed to successfully kick-start *a* blockchain application:

File: introduction.md [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+});               
+```
+
+Now, save and close `index.js` and try to start your newly created blockchain application by running:
+
+```bash
+node index.js # start the application
+```
+
+This should start the application with predefined default configurations, what will connect your app to a local Devnet.
```
Comment (RachBLondon) [inline introduction.md]: This should start the application with *the* predefined default configurations,*which* will connect your app to a local *devnet*.

File: introduction.md [hunk 12]
```diff
@@ -0,0 +1,170 @@
...
+From this point, you can start to [configure](configuration.md) and customize the application further.
+
+## Contributing to the Lisk SDK
+
+To test out your local changes of the Lisk SDK, use a blockchain application(e.g. [Lisk Core](../lisk-core/introduction.md)) as basis and link your locally customized `lisk-sdk`  to it.
```
Comment (RachBLondon) [inline introduction.md]: To test out your local changes of the Lisk SDK, use a blockchain application(e.g. [Lisk Core](../lisk-core/introduction.md)) as basis *, and link it to your locally customized `lisk-sdk`.*

File: introduction.md [hunk 13]
```diff
@@ -0,0 +1,170 @@
...
+
+1) At root of `lisk-sdk` repo,  run `npx lerna link`
```
Comment (RachBLondon) [inline introduction.md]: At root of *the* `lisk-sdk` repo,  run `npx lerna link`

File: introduction.md [hunk 14]
```diff
@@ -0,0 +1,170 @@
...
+2) At root level of your blockchain application, run `npm link lisk-sdk`
+
+*Notes:*
+1) If you have made changes in `elements` lib then make sure you run `npm run build` to see the changes
```
Comment (RachBLondon) [inline introduction.md]: If you have made changes in *the* `elements` *library* then make sure you run `npm run build` to see the changes

*If we are using lib else where then we can go for that, but I thought library was more consistent*

**Review by RachBLondon — CHANGES_REQUESTED** [review]
Some small grammatical changes, otherwise all good 👍

File: introduction.md (hunk 5 above)
Comment (Tschakki) [reply introduction.md]: Yes, ideally we should direct from here to the disclamie

**Review by RachBLondon — APPROVED** [review]

**Pull request merged**